### DISCUSSION TITLE
Welcome to xDrip Discussions!

### DISCUSSION THREAD

**Discussion by tolot27** [tolot27/xDrip#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DjDenmark** [comment]
Dear tolot27,

I'm a new user of xDrip (sub 14 days) and do already find the software amazing! I have a feature request regaring adding exercise as a treatment? The LibreLink app has 3 exercise intensities and then you can select the time exercised, which I find super informative when looking at my graphs. 
I hope this is the right place to write this comment. I have searched throughout the xDrip forum and cannot find a single post or comment regarding this feature. And I wanted to comment in the [official xDrip discussions #1900](https://github.com/NightscoutFoundation/xDrip/discussions/1900#discussion-3779078 ), but am not allowed because I'm not allowed, and I coulnd't find another way to contact you than this. 

So please help me submit the request in the correct place. I apologize for the inconvenience if I've commented in the wrong place. 

Thanks in advance, 
DjDenmark

**Reply by tolot27** [reply]
Hi @DjDenmark,

the correct place to ask you question is https://github.com/NightscoutFoundation/xDrip/discussions. You should be able to create a new discussion in the category [General](https://github.com/NightscoutFoundation/xDrip/discussions/categories/general).

**Reply by DjDenmark** [reply]
Thanks for the quick response! I've posted in the linked discussion.
